pages/application.html

```diff
@@ -4,6 +4,121 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
+    <link rel="stylesheet" href="../css/normalize.css" />
+    <link rel="stylesheet" href="../css/styles.css" />
+    <link rel="stylesheet" href="../css/application-styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
   </head>
-  <body></body>
+  <body>
+    <nav id="page-navigation" class="main-nav">
+      <ul>
+        <li>
+          <a href="/index.html"
+            ><img
+              src="../assets/svg/paw-solid.svg"
+              alt="paw-logo"
+              class="paw-logo"
+          /></a>
+        </li>
+        <div class="main-nav__flex">
+          <li><a href="about.html" class="f-s36">About us</a></li>
+          <li>
+            <a href="menu.html" class="f-s36">Menu</a>
+          </li>
+          <li>
+            <a href="application.html" class="f-s36">Careers</a>
+          </li>
+          <li>
+            <button class="primary-btn mr-60">
+              <a href="book-table.html" class="caps-text f-s32">Order now</a>
+            </button>
+          </li>
+        </div>
+      </ul>
+    </nav>
+    <div class="grid-container">
+      <form class="form-applicaion">
+        <h3 class="form-application-title">Job Application</h3>
+        <label for="firstname" class="form-applicaion-labelfirstname"
+          >First Name</label
+        >
+        <input
+          type="text"
+          id="firstname"
+          name="firstname"
+          class="form-applicaion-firstname"
+        />
+        <label for="lastname" class="form-applicaion-labelastname"
+          >Last Name</label
+        >
+        <input
+          type="text"
+          id="lastname"
+          name="lastname"
+          class="form-applicaion-lastname"
+        />
+        <label for="email" class="form-applicaion-labelemail">Email</label>
+        <input type="email" id="email" class="form-application-email" />
+        <label for="jobrole" class="form-applicaion-labeljobrole"
```

Comment:
Since we agreed to use BEM, I think your class names are not exactly following the methodology:
block-name__element-name (2 __ between block and element name, and if element is 2 words it has to be separated with - ) Please correct me if I am wrong.